### DISCUSSION TITLE
Name Column in EC2 Instances not seen randomly.

### DISCUSSION THREAD

**Discussion by maheshkoheda** [learndevopsonline/learndevopsonline#1276] [Q&A] [answered]
@r-devops Hi Raghu, EC2 Instances Interface sometimes randomly not showing name column and also i see there is no setting filter for this. How can i keep it visible all the time?
<img width="489" alt="Instance Name not seen sometimes randomly" src="https://github.com/learndevopsonline/learndevopsonline/assets/25144175/1fdb8276-d8b6-4215-91da-4b9726fe576d">

**Comment by Naresh-vk-243** [comment, marked as answer]
click on the settings icon on top right corner on instances page and try checking these boxes "tags" and select Name from tags dropdown

![image](https://github.com/learndevopsonline/learndevopsonline/assets/76612514/f03bafc0-a011-4619-9d4b-7b9a7ef40bd8)
